R/drive_publish.R

```diff
@@ -1,9 +1,15 @@
-#' Publish Google Drive file.
+#' Publish Google Drive file
 #'
-#' @template file
+#' Publish a Google Drive file to the web. You can only publish "Google" files, such
```

Comment:
Elsewhere we call these "native" or "Google native" files, I think. Regardless, let's be consistent.